test/e2e/crd_e2e_test.go

```diff
@@ -302,10 +304,15 @@ var _ = Describe("CRD Versions", func() {
 		sub, err := crc.OperatorsV1alpha1().Subscriptions(testNamespace).Get(context.TODO(), subscription.GetName(), metav1.GetOptions{})
```

Comment:
You don't need to do this. `Apply` handles it. you should be able to use the `subscription` variable declared above.

Reply:
Nice